javascript-methods/main.js

```diff
@@ -0,0 +1,53 @@
+const x = 10;
+const y = 20;
+const z = 30;
+const maximumValue = Math.max(x, y, z);
+console.log('maximumValue:', maximumValue);
+
+const heroes = [
+  'ironMan',
+  'hulk',
+  'blackWidow',
+  'thor'
+];
+// const random = Math.floor(Math.random(heroes.length - 1));
```

Comment:
Delete any old code that has no purpose!

Reply:
Thank you!